src/main/java/com/mraof/minestuck/data/recipe/MinestuckGristCostsProvider.java

```diff
@@ -319,96 +319,109 @@ protected void registerRecipes(Consumer<IFinishedRecipe> recipeSaver)
 		GristCostRecipeBuilder.of(MSItems.BLACKSMITH_HAMMER).grist(RUST, 8).grist(SULFUR, 9).grist(CAULK, 5).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.POGO_HAMMER).grist(BUILD, 20).grist(SHALE, 16).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.WRINKLEFUCKER).grist(BUILD, 32).grist(SHALE, 17).grist(TAR, 19).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.TELESCOPIC_SASSACRUSHER).grist(SHALE, 39).grist(TAR, 18).grist(MERCURY, 23).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.REGI_HAMMER).grist(AMETHYST, 25).grist(TAR, 70).grist(GOLD, 34).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.FEAR_NO_ANVIL).grist(BUILD, 999).grist(GARNET, 150).grist(DIAMOND, 54).grist(GOLD, 61).grist(QUARTZ, 1).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.MELT_MASHER).grist(BUILD, 1000).grist(TAR, 400).grist(GARNET, 200).grist(DIAMOND, 340).grist(GOLD, 100).grist(RUBY, 150).grist(SULFUR, 450).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.ESTROGEN_EMPOWERED_EVERYTHING_ERADICATOR).grist(BUILD, 8000).grist(SHALE, 1280).grist(URANIUM, 640).grist(RUST, 300).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.WRINKLEFUCKER).grist(BUILD, 55).grist(SHALE, 44).grist(TAR, 66).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.TELESCOPIC_SASSACRUSHER).grist(SHALE, 250).grist(TAR, 10).grist(MERCURY, 50).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.REGI_HAMMER).grist(AMETHYST, 15).grist(TAR, 20).grist(GOLD, 14).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.FEAR_NO_ANVIL).grist(BUILD, 5000).grist(GARNET, 5000).grist(DIAMOND, 5000).grist(GOLD, 5000).grist(QUARTZ, 1).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.MELT_MASHER).grist(BUILD, 8000).grist(TAR, 7800).grist(GARNET, 5000).grist(DIAMOND, 5000).grist(GOLD, 5000).grist(RUBY, 4000).grist(SULFUR, 6500).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.ESTROGEN_EMPOWERED_EVERYTHING_ERADICATOR).grist(BUILD, 8000).grist(SHALE, 2000).grist(URANIUM, 1500).grist(RUST, 2000).build(recipeSaver);
 		GristCostRecipeBuilder.of(MSItems.EEEEEEEEEEEE).grist(ARTIFACT, -100).grist(BUILD, 1).build(recipeSaver);
-		GristCostRecipeBuilder.of(MSItems.MWRTHWL).grist(BUILD, 10000).grist(GOLD, 1000).grist(RUST, 2500).build(recipeSaver);
-		
+		GristCostRecipeBuilder.of(MSItems.MWRTHWL).grist(BUILD, 40000).grist(GOLD, 10000).grist(RUST, 15000).grist(RUBY, 10000).grist(SHALE, 20000).grist(ZILLIUM, 1).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.ZILLYHOO_HAMMER).grist(BUILD, 28000).grist(GOLD, 15000).grist(URANIUM, 15000).grist(DIAMOND, 10000).grist(RUBY, 10000).grist(SHALE, 20000).grist(ZILLIUM, -1).build(recipeSaver);
+		GristCostRecipeBuilder.of(MSItems.POPAMATIC_VRILLYHOO).grist(BUILD, 28888).grist(COBALT, 18888).grist(URANIUM, 10000).grist(DIAMOND, 15000).grist(RUBY, 12500).grist(SHALE, 30000).grist(ZILLIUM, 1).build(recipeSaver);
```

Comment:
Zillium is really not fun at all to farm, so any more than one would be tedious. It might seem low, but getting any zillium grist requires a great deal more effort than normal grist types.